syntax = "proto2";
message TelemetryPacket {
	required int32 packet_id = 1;
	required fixed32 status = 2;
	required fixed32 timestamp = 3;
	optional float temperature_internal = 4;
	optional float temperature_external = 5;
	optional float altitude = 6;
	optional float latitude = 7;
	optional float longitude = 8;
	optional int32 pressure = 9;
	optional int32 bmp180_temperature = 10;
	optional float voltage = 11;
	optional float radiation = 12;
	optional uint32 ds18b20_temperature1 = 13;
	optional uint32 ds18b20_temperature2 = 14;
	optional uint32 ds18b20_temperature3 = 15;
	optional uint32 ds18b20_temperature4 = 16;
	optional int32 acceleration_x = 17;
	optional int32 acceleration_y = 18;
	optional int32 acceleration_z = 19;
	optional int32 gyroscope_x = 20;
	optional int32 gyroscope_y = 21;
	optional int32 gyroscope_z = 22;
	optional uint32 sun_sensor1 = 23;
	optional uint32 sun_sensor2 = 24;
	optional uint32 sun_sensor3 = 25;
	optional uint32 sun_sensor4 = 26;
}
